src/models/methods/index.ts

```diff
@@ -34,6 +35,8 @@ type Request = // account methods
              | AccountTxRequest
              | GatewayBalancesRequest
              | NoRippleCheckRequest
+               // payment channel methods
+             | ChannelVerifyRequest
```

Comment:
nit: move this to be in order with xrpl.org (so between path and order book methods and subscription methods)